### PR TITLE
docs: audit log provider

### PR DESCRIPTION
**Pull request by yildirayunlu**
Add auditLogProvider docs.

### CODE REVIEW

File: documentation/docs/core/providers/audit-log-provider.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    },
+};
+```
+
+#### Parameter Types
+
+| Name | Type    |
+| ---- | ------- |
+| id   | BaseKey |
+| name | string  |
```
Comment (aliemir) [inline documentation/docs/core/providers/audit-log-provider.md:221]:
```suggestion
| `id`   | `BaseKey` |
| `name` | `string`  |
```

**Review by aliemir — APPROVED** [review]

**Review by ozkalai — APPROVED** [review]

File: documentation/docs/core/hooks/audit-log/useLog.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+---
+id: useLog
+title: useLog
+---
+
+If you need to send a create audit log **refine** provides the `useLog` hook for it, It returns the `create` or `update` method from [`auditLogProvider`](/core/providers/audit-log-provider.md#create) under the hood.
```
Comment (salihozdemir) [inline documentation/docs/core/hooks/audit-log/useLog.md:6]:
```suggestion
If you need to create an audit log **refine** provides the `useLog` hook for it, It returns the `create` or `update` method from [`auditLogProvider`](/core/providers/audit-log-provider.md#create) under the hood.
```

File: documentation/docs/core/hooks/audit-log/useLog.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+});
+```
+
+## API
+
+### Properties
+
+| Property                                                                                            | Description                                              | Type                                                              | Default |
+| --------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------------------- | ------- |
+| <div className="required-block"><div>resource</div> <div className=" required">Required</div></div> | Resource name                                            | `string`                                                          |         |
```
Comment (salihozdemir) [inline documentation/docs/core/hooks/audit-log/useLog.md:41]:
```suggestion
| <div className="required-block"><div>resource</div> <div className=" required">Required</div></div> | Resource name                                            | `string`                                                          |  Resource name that it reads from URL       |
```

File: documentation/docs/core/hooks/audit-log/useLogList.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+
+## API
+
+### Properties
+
+| Property                                                                                            | Description                                              | Type                                                              | Default |
+| --------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | ----------------------------------------------------------------- | ------- |
+| <div className="required-block"><div>resource</div> <div className=" required">Required</div></div> | Resource name                                            | `string`                                                          |         |
```
Comment (salihozdemir) [inline documentation/docs/core/hooks/audit-log/useLogList.md:29]:
```suggestion
| <div className="required-block"><div>resource</div> <div className=" required">Required</div></div> | Resource name                                            | `string`                                                          |   Resource name that it reads from URL      |
```

File: documentation/docs/core/hooks/audit-log/useLog.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+| author                                                                                              | Author info                                              | `Record<string, any>`                                             |         |
+| meta                                                                                                | For make to unique                                       | `Record<string, any>`                                             |         |
+| data                                                                                                | Metadata query for `dataProvider`                        | [`MetaDataQuery`](/core/interfaces.md#metadataquery)              | {}      |
+| previousData                                                                                        | Callback to handle all related live events of this hook. | [`(event: LiveEvent) => void`](/core/interfaces.md#livemodeprops) |         |
+
+### Return values
+
+| Type |
+| ---- |
+| any  |
```
Comment (salihozdemir) [inline documentation/docs/core/hooks/audit-log/useLog.md:52]: I think, it should be update.
Additionally, we should categorize under `create` and `rename` that APIs.

File: documentation/docs/core/providers/audit-log-provider.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
+---
+id: audit-log-provider
+title: Audit Log Provider
+---
+
+## Overview
+
+Provides Audit Log support via **refine** `auditLogProvider`. A new log record is automatically created when a new record is created, updated and deleted. You can access these records from anywhere via hooks.
```
Comment (salihozdemir) [inline documentation/docs/core/providers/audit-log-provider.md:8]:
```suggestion
**refine** provides Audit Log support via `auditLogProvider`. A new log record is automatically created when a new record is created, updated and deleted. You can access these records from anywhere via hooks.
```

**Pull request closed without merging**